# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/editing/crashtests/inserttext-at-start-with-different-style.html
# Commit: 712bef3646ff
# Full Hash: 712bef3646ffbe11627b35c95d84c654ad1d6cea
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2022-01-31 21:28:43
# Description:
#   Bug 1752668 - Restore null-check in `HTMLEditor::ClearStyleAt` r=m_kato
#   
#   I don't remember why I removed the null-check in
#   https://phabricator.services.mozilla.com/D131748#change-Q0aewmYc3vaK
#   
# ==============================================================================

diff -r cfee576f2b42 -r 712bef3646ff testing/web-platform/tests/editing/crashtests/inserttext-at-start-with-different-style.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/tests/editing/crashtests/inserttext-at-start-with-different-style.html	Mon Jan 31 17:15:35 2022 +0000
@@ -0,0 +1,20 @@
+<html>
+<head>
+<meta charset="utf-8">
+<head>
+<script>
+function go() {
+  a.show();
+  document.execCommand("bold", false);
+  b.style.setProperty("font", "0px/43%");
+  document.execCommand("insertText", false, "a");
+}
+</script>
+</head>
+<body onload=go()>
+<ul contenteditable="true">
+<li id="b" style="font-weight: bolder">a</li>
+<dialog id="a" tabindex="0">a</dialog>
+</ul>
+</body>
+</html>
